torchaudio/models/wav2vec2/utils/import_huggingface.py

```diff
@@ -32,11 +32,17 @@ def _get_config(cfg):
 
 
 def _build(config, original):
+    if original.__class__.__name__ == 'Wav2Vec2ForCTC':
+        wav2vec2 = original.wav2vec2
+    else:
+        wav2vec2 = original
+
     imported = _get_model(**config)
```

Comment:
Note to reviewer

`original` here is an instance of either `Wav2Vec2Model` or `Wav2Vec2ModelForCTC`.
`Wav2Vec2ModelForCTC` instance contains an instance of `Wav2Vec2Model` as `wav2vec2` attribute.